src/js/lib/monster.apps.js

```diff
@@ -567,33 +567,19 @@ define(function() {
 			var self = this,
 				loadDefaultLanguage = _.bind(self.loadLocale, self, app, monster.defaultLanguage),
 				maybeLoadPreferredLanguage = function maybeLoadPreferredLanguage(app, language, callback) {
-					/**
-					 * The requested language is the same as the default so skip load again
-					 */
 					if (language.toLowerCase() === monster.defaultLanguage.toLowerCase()) {
 						return callback(null);
 					}
-
-					/**
-					 * The selected language is not supported so we set the default for the app
-					 */
 					if (!_.has(app.i18n, language)) {
-						console.info(language + ' isn\'t a supported language by this application: ' + app.name + ' switching automatically to ' + monster.defaultLanguage);
-						self.loadLocale(
-							app,
-							monster.defaultLanguage,
-							// Prepend null as we don't care if it errors out, the app can still load
-							_.partial(callback, null)
-						);
-
```

Comment:
I think it is not necessary to pass `{ languageSupported: false }`

Se the other comment for details.